poc_mobile_verifier/src/subnetwork_rewards.rs

```diff
@@ -66,13 +66,14 @@ impl SubnetworkRewards {
         self,
         subnet_rewards_tx: &file_sink::MessageSender,
     ) -> file_store::Result<oneshot::Receiver<file_store::Result>> {
-        file_sink::write(
+        file_store::write!(
+            "subnet_rewards",
```

Comment:
unless you want to take some conditional extra load into the macro that preps the prometheus label we may want to rethink the `module_path!()` as a component of the label; in a situation like this it would produce the label `poc_mobile_verifier_subnetwork_rewards_subnet_rewards`; it's probably still a good strategy if the macro either allows for the label to be optional altogether (but would rely on implicit understanding of the naming strategy which the macro is hiding) or else conditionally lop off the last element of the path when it matches the supplied suffix.

the name of the calling app plus the supplied tag would probably be a good happy medium if this gets overly redundant

Reply:
I made it optional. But, the more I think about it - your happy medium of crate+label seems best - no implicit magic to worry about and tags remain stable after refactoring (assuming crate names won't change much, of course).

Reply:
@jeffgrunewald Switched it to the happy medium.